front/src/common/Map/Search/MapSearchStation.tsx

```diff
@@ -69,9 +80,10 @@ const MapSearchStation = ({ updateExtViewport, closeMapSearchPopUp }: MapSearchS
       });
   }, [searchState]);
 
-  const searchByNames = useCallback(async () => {
-    const searchQuery = ['search', ['name'], searchState];
+  const searchByCodesAndNames = useCallback(async () => {
```

Comment:
Same for this function and the next one, it doesn't seem to be heavy, not sure a useCallback is necessary